### PR TITLE
Added Card Example for products and profile

### PR DESCRIPTION
**Pull request by ruhulmus**
Added Bootstrap Card example for 
1. E-commerce Products list with Add to cart button.
2. Users Profile

### CODE REVIEW

File: site/content/docs/5.1/examples/card/index.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      <span class="fs-4">Product and Profile Card</span>
+    </a>
+  </header>
+
+  <main>
+    <h2>Product Card</h2>
+    <div class="row flex-nowrap justify-content-between align-items-center mb-5">
+      <div class="col-4 pt-1">
+          <div class="card">
+            <img
```
Comment (WinterSilence) [inline site/content/docs/5.1/examples/card/index.html:21]: you can use [placeholders](https://getbootstrap.com/docs/5.1/components/placeholders/)

File: site/content/docs/5.1/examples/card/card.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.card-main{
```
Comment (WinterSilence) [inline site/content/docs/5.1/examples/card/card.css:1]: see Card section [Background and color](https://getbootstrap.com/docs/5.1/components/card/#background-and-color)

File: site/content/docs/5.1/examples/card/index.html (hunk 1 above)
Comment (ruhulmus) [reply site/content/docs/5.1/examples/card/index.html:22]: @WinterSilence, Thanks for your feedback. I was added the `<img` tag to use a custom image as a placeholder. 
By the way, I have added the bootstrap placeholders as well. Please review it now.

File: site/content/docs/5.1/examples/card/card.css (hunk 2 above)
Comment (ruhulmus) [reply site/content/docs/5.1/examples/card/card.css:1]: @WinterSilence, Thanks again for the suggestion. The main reason was to use custom class and CSS for it. 
BTW, Now I have updated these sections with bootstrap background and color. So now both are exist (custom class + Bootstrap class)

File: site/content/docs/5.1/examples/card/index.html (hunk 1 above)
Comment (WinterSilence) [reply site/content/docs/5.1/examples/card/index.html:22]: also, void tags in HTML5 don't use slash at end: `<tag attr="valie"/>` - invalid, `<tag attr="valie">` - valid

File: site/content/docs/5.1/examples/card/index.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    <a href="/" class="d-flex align-items-center text-dark text-decoration-none">
+      <svg xmlns="http://www.w3.org/2000/svg" width="40" height="32" class="me-2" viewBox="0 0 118 94" role="img"><title>Bootstrap</title><path fill-rule="evenodd" clip-rule="evenodd" d="M24.509 0c-6.733 0-11.715 5.893-11.492 12.284.214 6.14-.064 14.092-2.066 20.577C8.943 39.365 5.547 43.485 0 44.014v5.972c5.547.529 8.943 4.649 10.951 11.153 2.002 6.485 2.28 14.437 2.066 20.577C12.794 88.106 17.776 94 24.51 94H93.5c6.733 0 11.714-5.893 11.491-12.284-.214-6.14.064-14.092 2.066-20.577 2.009-6.504 5.396-10.624 10.943-11.153v-5.972c-5.547-.529-8.934-4.649-10.943-11.153-2.002-6.484-2.28-14.437-2.066-20.577C105.214 5.894 100.233 0 93.5 0H24.508zM80 57.863C80 66.663 73.436 72 62.543 72H44a2 2 0 01-2-2V24a2 2 0 012-2h18.437c9.083 0 15.044 4.92 15.044 12.474 0 5.302-4.01 10.049-9.119 10.88v.277C75.317 46.394 80 51.21 80 57.863zM60.521 28.34H49.948v14.934h8.905c6.884 0 10.68-2.772 10.68-7.727 0-4.643-3.264-7.207-9.012-7.207zM49.948 49.2v16.458H60.91c7.167 0 10.964-2.876 10.964-8.281 0-5.406-3.903-8.178-11.425-8.178H49.948z" fill="currentColor"></path></svg>
```
[the remark this answers is not in the document]
Comment (ruhulmus) [reply site/content/docs/5.1/examples/card/index.html:11]: I have got the same code on several examples. like - `starter-template`, `jumbotron`, `jumbotron` etc.
I have updated the valid `viewBox` sections.

[the remark this answers is not in the document]
Comment (WinterSilence) [reply site/content/docs/5.1/examples/card/index.html]: @ruhulmus you must complete learning HTML5/CSS3 before trying change anything.

[the remark this answers is not in the document]
Comment (ruhulmus) [reply site/content/docs/5.1/examples/card/index.html]: I already knew it and completely understand. but the thing is, You can see already added the `role="img"` in the other sections. 
The reason behind usage here is that many examples use this on the `<svg` sections too. it's not restricted to adding on the SVG sections.

[the remark this answers is not in the document]
Comment (WinterSilence) [reply site/content/docs/5.1/examples/card/index.html]: @ruhulmus if I'm ask you "role="img" where you find this?" it's only question. I'm don't see this role before and [explode it now](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role).

[the remark this answers is not in the document]
Comment (ruhulmus) [reply site/content/docs/5.1/examples/card/index.html]: Its recommended and good practice to use role="img". I thought you mean that is not standard to use.. and ask me for reason. your question was not clear to me.  maybe it's my bad.

[the remark this answers is not in the document]
Comment (WinterSilence) [reply site/content/docs/5.1/examples/card/index.html]: @ruhulmus I'm don't seen this role before. I asked you to understand: you taken this value from manual or used custom value (invalid).

[the remark this answers is not in the document]
Comment (ruhulmus) [reply site/content/docs/5.1/examples/card/index.html]: > @ruhulmus I'm don't seen this role before. I asked you to understand: you taken this value from manual or used custom value (invalid).

It was an understanding of your question :). I did not get it clearly.

**Comment by mdo** [comment]
This example is incomplete and not up to the same standards as the rest of our examples. You're welcome to keep iterating and present your ideas again in the future, but for now, I'm going to close this.

**Comment by WinterSilence** [comment]
@ruhulmus he tried told you: use placeholders like as `<svg class="bd-placeholder-img ..." width="100%" height="..." xmlns="http://www.w3.org/2000/svg" role="img" aria-label="..." preserveAspectRatio="xMidYMid slice" focusable="false"><title>...</title><rect width="100%" height="100%" fill="#..."></rect></svg>` instead `img` tags

**Pull request closed without merging**